fedimint-client/src/lib.rs

```diff
@@ -1758,7 +1758,7 @@ impl ClientBuilder {
             module_inits: client.module_inits.clone(),
             primary_module_instance: Some(client.primary_module_instance),
             admin_creds: None,
-            db: client.db.clone(),
+            db_no_encoders: client.db.with_decoders(Default::default()),
```

Comment:
This function is eventually going away right? Initially just confused me since this is called `db_no_encoders` and its created using `with_decoders`

Reply:
> `from_existing` is eventually going away right? 

No idea. If so, good riddance. :)

Reply:
Yep, commented that in the other PR, we don't need it anymore now that the client can shut down cleanly, dropping all DB references.